monit/config/monit.conf

```diff
@@ -0,0 +1,319 @@
+###############################################################################
```

Comment:
If you want these comments to show up when you run `hab pkg config` you should move them to the default.toml

Reply:
For this, already has the same comment here: https://github.com/habitat-sh/core-plans/pull/902#discussion_r147506406 what should we do then? :)